app/assets/stylesheets/_simple-sidebar.scss

```diff
@@ -128,3 +137,12 @@
 .sidebar-icon {
   margin-right: 15px;
 }
+
+/* Settings Sidebar */
+.sidebar-item.settings-sidebar-item {
```

Comment:
Looks like there's a lot of style spread in this file, including the ones using `>`, I think there's something missing in the organization here. Also we should not depend on the `sidebar-item` to apply this style, why not just use`settings-sidebar-item` here?